app/models/supabase_client.py

```
@@ -354,7 +357,7 @@ def get_data(
         try:
             response = query.execute()
         except Exception as e:
-            print("SupabaseClient get_data error:", e, flush=True)
+            logger.exception(f"[get_data] SupabaseClient get_data error: {e}")
```

Ruff (0.13.3)

Redundant exception object included in `logging.exception` call

(TRY401)